ontology/kisao.md

```diff
@@ -24,6 +29,48 @@ license:
 activity_status: active
 repository: https://github.com/SED-ML/KiSAO
 preferredPrefix: KISAO
+usages:
+  - user: https://sed-ml.org/
+    type: annotation
+    description: The Simulation Experiment Description Markup Language (SED-ML) is a language for describing simulations and visualizations of their results.
+    examples:
+      - url: https://sed-ml.org/examples.html
+        description: "Several examples of simulations encoded in SED-ML"
+  - user: https://biosimulations.org/
+    type: annotation
+    description: BioSimulations is a repository of biosimulation projects.
+    examples:
+      - url: https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000
+        description: "Simulation of a synthetic oscillatory biochemical network"
+  - user: https://run.biosimulations.org/
+    type: annotation
+    description: runBioSimulations is a web application for execution biological simulations.
+    examples:
+      - url: https://run.biosimulations.org/runs?try=1
```

Comment:
Can you detail how KISAO ids can be witnessed on this website?

Reply:
This application uses KiSAO to report which simulation algorithm was actually executed and with what parameter values. For various reasons, this is important to make transparent.

Names of KiSAO concepts are displayed 1 level deeper such as at https://run.biosimulations.org/runs/61fea483f499ccf25faafc4d#tab=log. This example shows that the simulation was executed using synchronous logical model simulation method (KISAO_0000449).

Reply:
Indeed, thank you for the clarification